srcpkgs/vmdfmt/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'vmdfmt'
+pkgname=vmdfmt
+version=0.0.2
+revision=1
+wrksrc="vmd-${version}"
+build_style=go
+go_import_path="github.com/bobertlo/vmd"
+go_package="github.com/bobertlo/vmd/cmd/vmdfmt"
+hostmakedepends="git"
```

Comment:
This implies that your dependencies aren't stable, and indeed looking at your source repo the build is non-deterministic w.r.t. dependencies.  Please manage your dependencies with some kind of tool like `dep`, or use Go Modules (but I recommend `dep` in this case since its more stable and more automatic).

Reply:
You now need a pre_build() that actually uses dep to get your dependencies.

Reply:
Oh, I wasn't paying attention.  If you've vendored them into the tree that's fine, and it can just be accepted as is.

Reply:
Oh, cool. It's only two packages and it was a little unclear reading around about what to do!